kafka/init.lua

```diff
@@ -359,6 +386,36 @@ function Producer:dump_conf()
     return self._producer:dump_conf()
 end
 
+function Producer:metadata(options)
+    if self._producer == nil then
+        return
+    end
+
+    local timeout_ms = DEFAULT_TIMEOUT_MS
+    if options ~= nil and options.timeout_ms ~= nil then
+        timeout_ms = options.timeout_ms
+    end
+
+    local topic
+    if options ~= nil and options.topic ~= nil then
+        topic = options.topic
+    end
+
+    return self._producer:metadata(topic, timeout_ms)
+end
+
+function Producer:list_groups(group, timeout_ms)
```

Comment:
The same comment: maybe it's better to use `options` style for `timeout` and probably `group` options

Reply:
Sounds reasonable. Fixed.